docs/operate/customize/look-and-feel/override_components.md

```diff
@@ -39,7 +39,14 @@ You can always [ask for help](../../../install/troubleshoot.md#getting-help)!
 
 ### Find the React Overridable ID
 
-The React component's overridable id for the '`Metadata-only record`' checkbox component is `ReactInvenioDeposit.FileUploaderToolbar.MetadataOnlyToggle.container`. It can be found in the [react-invenio-deposit](https://github.com/inveniosoftware/react-invenio-deposit/blob/040b698967e7fee14ac7fe466e49f883efbe8c47/src/lib/components/FileUploader/FileUploaderToolbar.js#L55) module.
+The easiest way to to find the ID of an overridable component is to use `react-overridable`'s built-in developer tool.
+Simply open a browser console on your local instance and call the global function `reactOverridableEnableDevMode()`.
+All overridable components will display a small red overlay tag showing their ID.
+You can click a tag to copy its ID to your clipboard.
+
+![Metadata-only checkbox overridable ID in an overlay](./imgs/metadata_id_overlay.png)
+
+The React component's overridable ID for the '`Metadata-only record`' checkbox component is `ReactInvenioDeposit.FileUploaderToolbar.MetadataOnlyToggle.container`. It can be found in the [react-invenio-deposit](https://github.com/inveniosoftware/react-invenio-deposit/blob/040b698967e7fee14ac7fe466e49f883efbe8c47/src/lib/components/FileUploader/FileUploaderToolbar.js#L55) module.
```

Comment:
@palkerecsenyi this module is deprecated. We moved the code in [invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/dd72962b713f07b81699f7d5c9a8a673d585466a/invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/FileUploader/FileUploaderToolbar.js#L56). Can you update the link? If there is any visual when you copy the id maybe update the screenshot if it is easy.